src/solver/types.jl

```diff
@@ -439,6 +403,10 @@ function AtmosConfig(;
         @info "Using default configuration"
         default_config_dict()
     end
+    return AtmosConfig(config; comms_ctx)
+end
+
+function AtmosConfig(config::Dict; comms_ctx = nothing)
```

Comment:
It would be useful to add docstrings for these functions since they are user-facing.